### PR TITLE
return std::map of std::shared_ptr for getVFPxTables()

### PR DESCRIPTION
**Pull request by GitPaean**
in Schedule.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
The downstream PR is https://github.com/OPM/opm-simulators/pull/1462 .

**Comment by GitPaean** [comment]
jenkins build this opm-simulators=1462 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=1462 please

File: opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp [hunk 1]
```diff
@@ -111,8 +111,8 @@ namespace Opm
         bool hasOilVaporizationProperties() const;
         const VFPProdTable& getVFPProdTable(int table_id, size_t timeStep) const;
         const VFPInjTable& getVFPInjTable(int table_id, size_t timeStep) const;
-        std::map<int, VFPProdTable> getVFPProdTables(size_t timeStep) const;
-        std::map<int, VFPInjTable>  getVFPInjTables(size_t timeStep) const;
+        std::map<int, std::shared_ptr<const VFPProdTable> > getVFPProdTables(size_t timeStep) const;
+        std::map<int, std::shared_ptr<const VFPInjTable> > getVFPInjTables(size_t timeStep) const;
```
Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp]: Looks like a missing `>`?

**Pull request merged**